src/components/forms/footer/footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {StyledFooter, StyledLink, StyledText} from "./footer.styled";
+
+const Footer = ({text, path, link}) => {
+    return (
+        <StyledFooter>
+            {text && <StyledText>{text}</StyledText>}
```

Comment:
You have text all the time in this component why do you need to track it text && ...?